pkg/database/migrations.go

```diff
@@ -1801,6 +1801,35 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
 				return nil
 			},
 		},
+		{
+			ID: "00075-AddAlternateSMSTemplates",
+			Migrate: func(tx *gorm.DB) error {
+				sqls := []string{
+					"ALTER TABLE realms ADD COLUMN IF NOT EXISTS alternate_sms_templates text[]",
```

Comment:
We should probably use a single hstore column for the alternates instead

Reply:
Nice. Postgres has everything.